src/core/reducers/addons.js

```diff
@@ -41,15 +41,17 @@ export type AddonsState = {|
   infoBySlug: {
     [slug: string]: {| info: AddonInfoType, loading: boolean |},
   },
-  loadingBySlug: { [addonSlug: string]: boolean },
+  byIdInURL: { [addonSlug: string]: AddonID },
```

Comment:
Should this be renamed to `[addonId: string]`?

Reply:
probably yes